examples/create-and-initialize-hpcs/main.tf

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Can we name it complete for this example - just to keep the consistency?

Reply:
Good point. Updated.